quickstart/docker-compose.yaml

```diff
@@ -21,22 +22,26 @@ services:
       KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE: "true"
 
   kafka-init:
-    image: docker.io/bitnami/kafka:latest
+    image: docker.io/bitnami/kafka:3.2
+    container_name: temp-kafka-init-zq
     command: 
       - "/bin/bash"
       - "-c"
       -  | 
         /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic items-crud --config cleanup.policy=compact
         /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic events-sse
         /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic echo-service-messages
-        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic route-guide-requests
-        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic route-guide-responses
+        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic route-guide-requests --config cleanup.policy=delete
+        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic route-guide-responses --config cleanup.policy=delete
```

Comment:
Default policy is `delete`, so no need to specify?

Reply:
Akram added this last weekend when we were doing testing. Is it no longer needed after some updates?

Reply:
@akrambek do we still need this?